jenkins/ompi/Dockerfile.centos7

```diff
@@ -0,0 +1,46 @@
+ARG OMPI_CI_OS_NAME=centos
+ARG OMPI_CI_OS_VERSION=7
+FROM ${OMPI_CI_OS_NAME}:${OMPI_CI_OS_VERSION}
+
+RUN yum install -y \
+    atk \
+    bc \
+    binutils-devel \
+    bzip2 \
+    cairo \
+    environment-modules \
+    flex \
+    gcc \
+    gcc-c++ \
+    gcc-gfortran \
+    git \
+    gtk2 \
+    jq \
+    libmnl \
+    libnl3 \
+    make \
+    numactl-devel \
+    numactl-libs \
+    perl \
+    perl-Data-Dumper \
+    perl-Thread-Queue \
+    sudo \
+    tcsh \
+    tk \
+    valgrind-devel \
+    wget
+
+# Engineering versions of Mellanox HPC-X components are loaded from the Mellanox lab environment modules
+RUN echo "/hpc/local/etc/modulefiles" >> /usr/share/Modules/init/.modulespath
+
+ARG OMPI_CI_MOFED_VERSION=4.7-1.0.0.1
+# TODO: 'rhel7.6' is hardcoded for centos7
```

Comment:
It can be a variable defined close to CentOS tag.

Reply:
Could you please provide an example how you see it?